projects/fusion-ui/entities/layout-user.ts

```diff
@@ -1,5 +1,5 @@
 export interface LayoutUser {
-    icon?: string;
+    icon?: string | {iconName: string; iconVersion: string};
```

Comment:
add type `string | {iconName: string; iconVersion: string}` in icon component and export it out in all places